NOTICES.txt

```diff
@@ -0,0 +1,467 @@
+=============== TABLE OF CONTENTS =============================
+
+
+The following is a listing of the cyberark/conjur module's open source components
+detailed in this document. This list is provided for your convenience; please read
+further if you wish to review the copyright notice(s) and the full text of the
+license associated with each component.
+
+Section 1: Apache License 2.0
+
+>>> github.com/puppetlabs/puppet-6.0.10
+>>> github.com/puppetlabs/puppetlabs-registry-3.1.0
+
+
+Section 2: BSD 2-clause "Simplified" License
+
+>>> github.com/ruby/ruby-2.4.10
+
+
+
```

Comment:
there should be an appendix with the full template licenses for those included in this file (some license types require that we share the full license text)
```

>>> Apache License 2.0

>>> BSD 2-clause "Simplified" License